packages/react/package.json

```diff
@@ -25,7 +25,7 @@
   "exports": {
     ".": {
       "import": {
-        "types": "./dist/index.d.mts",
+        "types": "./dist/index.d.ts",
```

Comment:
Core of the problem, `tsc` can't dual emit `.d.ts` as CJS and `.d.mts` as ESM so it's only `.d.ts` for both